doc/nrf/libraries/zigbee/zigbee_fota.rst

```diff
@@ -8,7 +8,7 @@ Zigbee FOTA
    :depth: 2
 
 The Zigbee firmware over-the-air (Zigbee FOTA) library provides Zigbee endpoint definition, which implements clusters responsible for transferring a firmware file through the Zigbee network.
-The received data is passed as an upgrade candidate through the :ref:`lib_dfu_target` library API.
+The received data is passed as an upgrade candidate through the :ref:`lib_dfu_multi_image` library API to the :ref:`lib_dfu_target`.
```

Comment:
Documentation build fails because this tag isn't present before [#7493](https://github.com/nrfconnect/sdk-nrf/pull/7493) is merged. You'll need to rebase once that goes in.

Reply:
rebased! :)